python/pp_ser.py

```diff
@@ -121,8 +121,7 @@ def __init__(self, infile, outfile='', ifdef='SERIALIZE', real='ireals',
         self.__module = ''       # current module
         self.__calls = set()     # calls to serialization module
         self.__outputBuffer = '' # preprocessed file
-        self.__implicitnone_found = False # has a implicit none statement been found?
-        self.__implicitnone_done = False # has code been inserted at IMPLICIT NONE?
+        self.__useStmtInModule = False  # USE statement was inserted in module
```

Comment:
@pspoerri sure, I was not aware of this rules ... pp_ser in kind of my first real Python "project" :-)